TestBenches/ProjectionRouter_test.cpp

```diff
@@ -28,47 +31,12 @@ int main()
   const string vmProjectionPattern = "VMProjections*";
 
   // Define region according to which layer is being tested
-#if MODULE_ == PR_L1PHIB_ || MODULE_ == PR_L2PHIB_ || MODULE_ == PR_L3PHIB_ || MODULE_ == PR_L1PHIC_ || MODULE_ == PR_L2PHIC_ || MODULE_ == PR_L3PHIC_
-  const auto projMemType = BARRELPS;
-#elif MODULE_ == PR_L4PHIB_ || MODULE_ == PR_L5PHIB_ || MODULE_ == PR_L6PHIB_ || MODULE_ == PR_L4PHIC_ || MODULE_ == PR_L5PHIC_ || MODULE_ == PR_L6PHIC_
-  const auto projMemType = BARREL2S;
-#else
-  #error "Undefined Module"
-#endif
-
-#if MODULE_ == PR_L1PHIB_ || MODULE_ == PR_L2PHIB_ || MODULE_ == PR_L3PHIB_ || MODULE_ == PR_L4PHIB_ || MODULE_ == PR_L5PHIB_ || MODULE_ == PR_L6PHIB_ || MODULE_ == PR_L1PHIC_ || MODULE_ == PR_L2PHIC_ || MODULE_ == PR_L3PHIC_ || MODULE_ == PR_L4PHIC_ || MODULE_ == PR_L5PHIC_ || MODULE_ == PR_L6PHIC_
+  assert(MODULE_ >= 413 && MODULE_ <= 440); // Select for PR modules
+  assert(std::string(module_name[MODULE_]).find("PHIB") != string::npos ||
+         std::string(module_name[MODULE_]).find("PHIC") != string::npos); // Select for PHIB and PHIC modules
+  const auto projMemType = (MODULE_ >= 413 && MODULE_ <= 428) ? BARRELPS : BARREL2S;
```

Comment:
@aperloff Is there any reason we can't replace the magic numbers here with the members of the `Module` enum? For example:
```c++
  assert(MODULE_ >= PR_L1PHIA_ && MODULE_ <= PR_L6PHID_); // Select for PR modules
```

Reply:
Fixed